frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx

```diff
@@ -897,6 +909,27 @@ export const AlertConfigurationCard = ({
 							</Form.Item>
 						</section>
 
+						<section>
+							<h3>Webhooks to Notify</h3>
+							<p>
+								Add webhook destinations for this alert, sent as
+								JSON over HTTP. See the{' '}
+								<Link to="https://www.highlight.io/docs/general/product-features/general-features/alerts/webhooks">
```

Comment:
Assuming these docs will be released?

Reply:
yup in https://github.com/highlight/highlight.io/pull/455